core/modules/ckeditor5/ckeditor5.install

```diff
@@ -31,3 +31,22 @@ function ckeditor5_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Updates the editor settings key from "cketoolbar" to just "toolbar".
+ */
+function ckeditor5_update_1000() {
+  // cspell:ignore cketoolbar
+  $config_names = config_get_names_with_prefix('filter.format');
+  foreach ($config_names as $config_name) {
+    $config = config($config_name);
+    if ($config['editor'] === 'ckeditor5' && isset($config['editor_settings']['cketoolbar'])) {
+      $editor_settings = $config['editor_settings'];
+      $toolbar = $editor_settings['cketoolbar'];
+      unset($editor_settings['cketoolbar']);
+      $config['editor_settings'] = array_merge(array(
```

Comment:
Wouldn't $config->set() and $config->clear() be slightly more readable?

And where's $config->save()?

Reply:
Thanks @indigoxela, I don't know what I was thinking here. Maybe I accidentally missed a commit or something? There's no way this would work as-is. I'll rework it.

Reply:
Updated commit that should save properly.

> Wouldn't $config->set() and $config->clear() be slightly more readable?

This approach keeps the config keys in the expected order. Using `clear()` and `set()` would push the new `toolbar` key to the bottom of the array of settings instead of as the first item.